src/App.js

```diff
@@ -1,6 +1,9 @@
 import './App.css';
 import Home from './Components/Home/Home';
 import { Routes, Route } from "react-router-dom";
+import Login from './Components/Authentication/Login';
+import Signup from './Components/Authentication/Signup';
+import ProductListing from './Components/Product/ProductListing';
```

Comment:
Can export all component from a common file like index.js and then import from 1 file, makes code more cleaner and well structured.

Reply:
Resolved this- created a common file ComponentImporter to export the components and then imported them in App.js